user_data.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
```

Comment:
Why is the user-data file here ? This should be used by the Launch configuration of the Autoscaling group, which is another folder.